elements/pfe-modal/src/pfe-modal.js

```diff
@@ -1,3 +1,6 @@
+// Import polyfills: filter
+import "./polyfills--pfe-modal.js";
```

Comment:
@castastrophe - Should this be pulled out into a separate PR? Was it causing the e2e to fail?

Reply:
It was causing e2e to fail in IE11 for pfe-modal. We can still pull it out though if that's easier.

Reply:
Let's just leave it in.